### PR TITLE
Vet-clinic database: Query and update

### PR DESCRIPTION
**Pull request by amare1990**
- [ ] Added `species` filed to the animals table.
- [ ] Updated the species for all animals in the animals table.
- [ ] Used `min, max, count, avg, group by` aggregate sql statements.
- [ ] Exercised on Postgres transaction.
- [ ] Updated README.md file.

**below are screenshots for most sql statements**
Screenshot from 2022-12-13 23-39-31
Screenshot from 2022-12-13 23-38-35
Screenshot from 2022-12-13 23-38-01
Screenshot from 2022-12-13 23-36-43
Screenshot from 2022-12-13 23-36-05
Screenshot from 2022-12-13 23-35-09
Screenshot from 2022-12-13 22-28-27
Screenshot from 2022-12-13 22-04-56
Screenshot from 2022-12-13 21-56-23
Screenshot from 2022-12-13 21-25-08
Screenshot from 2022-12-13 20-54-27

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -10,3 +10,57 @@ select * from animals where name != 'Gabumon';
...
 
+/* On Query and Update animals table: second milestone of the project */
+
+begin;
+update animals
+set species = 'unspecified';
+select * from animals;
+rollback;
+
+
```
Comment (Munsa1) [inline queries.sql]: - [ ] After rollback add a select statement as done in line `18` to verify the change was made

File: queries.sql [hunk 2]
```diff
@@ -10,3 +10,57 @@ select * from animals where name != 'Gabumon';
...
+BEGIN;
+UPDATE animals
+SET species = 'digimon'
+WHERE species LIKE '%mon';
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+
+SELECT * FROM animals;
```
Comment (Munsa1) [inline queries.sql]: - [ ] Again here select the species column as it is the column that was changed to verify the change made.as this is a requirement 
- [ ] Also after verifying hte change kindly make a `commit` as this is part of the requirements
- [ ] Then lastly select the species column again to verify the change was made again as this is a requirement.

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @amare1990, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by addod19 — APPROVED** [review]
### Hi @amare1990, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Highlights
- [ ] Nice job addressing all the comments made by the previous code reviewer :100: 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**